src/Uno.UI.Runtime.Skia/GLCanvasElement.cs

```diff
@@ -0,0 +1,190 @@
+using System.Runtime.InteropServices;
+using Windows.Foundation;
+using Microsoft.UI.Composition;
+using Silk.NET.OpenGL;
+using SkiaSharp;
+using Uno.Foundation.Extensibility;
+using Uno.UI.Composition;
+using Uno.UI.Runtime.Skia;
+
+namespace Microsoft.UI.Xaml.Controls;
+
+public abstract class GLCanvasElement : FrameworkElement
+{
+	private class GLVisual(GLCanvasElement owner, Compositor compositor) : Visual(compositor)
```

Comment:
Let's move to a separate file (`GLCanvasElement.GLVisual.cs` if it needs to stay nested)